lane.py

```diff
@@ -2868,80 +2591,6 @@ def search(self, _db, query_string, search_client, pagination=None,
         return m(_db, query_string, search_client, pagination,
                  facets=facets)
 
-    def bibliographic_filter_clause(self, _db, qu, featured, outer_join=False):
```

Comment:
This whole thing was merged into `DatabaseBackedWorkLane.bibliographic_filter_clauses` (it returns multiple clauses rather than one joined clause because that's easier to test).